### ISSUE TITLE
Allow saving specifc searches as Custom Feeds (or maybe Personal Feeds)

### ISSUE THREAD

**Issue by adamatic-me** [bluesky-social/social-app#2560]
**Describe the solution you'd like**

I would like to be able to save a search so that new results for those saved searches continue to appear in my feeds in the future.

On https://bsky.app/search display a Create Feed button.

On https://bsky.app/feeds and https://bsky.app/settings/saved-feeds display a new section called Custom Feeds, or similar. (Maybe Personal Feeds?)
User can delete feeds here. Maybe can also create a new Custom Feed here as well.

example flow: 

1. User searches: https://bsky.app/search?q=exoplanet 
2. User clicks Create Feed button on the search page
3. User is taken to something like: https://bsky.app/profile/did:plc:sadsad3dfssfd/feed/exoplanet

**Comment by pfrazee** [comment]
Yeah I quite like this idea and hope it's something we can introduce in the future.

**Comment by soags** [comment]
Great idea.
If combined with "Advanced Search", it would work as a user-friendly feed generator.
I think 90% of published feeds can be handled by keywords and `from:username`, `to:username`, `regex://`, `since:`, `like:>10`, etc.

**Comment by chris-mcdonald-dev** [comment]
Any updates on this?

**Comment by chris-mcdonald-dev** [comment]
@pfrazee Hate to tag you, but just circling back a year later. Any updates on discussions around this?

**Comment by systemaddict** [comment]
I would also really encourage this. I've created a few custom feeds that have become a bit popular around and people constantly ask me how to tweak it. I think 95% of all the wishes could be accommodate with a simple advanced search -> custom feed.

**Comment by ljredux** [comment]
Yeah, I want this badly.
